.github/workflows/ci.yml

```diff
@@ -20,8 +20,8 @@ on: push
 # - standardize WSL tasks by using GitHub-provided runners
 
 env:
-  CI_IMAGE: registry.gitlab.com/buildstream/buildstream-docker-images/testsuite-fedora:30-master-154893333
-  PYTEST_ARGS: --color=yes --integration -n 4
```

Comment:
I had this problem with bst-plugins-experimental, if it's managing to complete the tests within the given free time allocation then I guess it's fine (even if we have to wait...)

Reply:
Nod, we can restore it if we figure out what's wrong, it's a bit slow but not impossible to work with.